Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -79,15 +128,28 @@
     'vtPositives': 10,
     'vtPositiveUrlsForIP': 30
 }
-# The dictionary below does not represent DBot Scores correctly, and should not be used
-dbotscores = {
-    'Critical': 4,
-    'High': 3,
-    'Medium': 2,
-    'Low': 1,
-    'Unknown': 0,
-    'Informational': 0.5
-}
+
+
+class DBotScoreType(object):
+    """ ignore docstring
```

Comment:
ignore docsting?

Reply:
I added in the common_server_docs.py condition that if the docstring contains `ignore docstring` it will ignore generating docs for it.
Till now if user wanted to ignore/skip function from being in Script Helper documentation then he had to add it under `pyPrivateFuncs`.

Reply:
I deleted all the ignore docstring, instead I will generate docstring for classes too